osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SliderBody.cs

```diff
@@ -21,8 +21,8 @@ public abstract class SliderBody : CompositeDrawable
 
         public float PathWidth
```

Comment:
Was it a conscious decision to not rename this as well?

Reply:
no, this is 100% oversight